src/main/scala/nl/knaw/dans/easy/stage/package.scala

```diff
@@ -18,11 +18,14 @@ package nl.knaw.dans.easy
 import java.net.{HttpURLConnection, URL}
 
 import nl.knaw.dans.pf.language.ddm.handlermaps.NameSpace
-import nl.knaw.dans.pf.language.emd.types.EmdConstants
 
-import scala.util.Try
+import scala.util.{Failure, Success, Try}
 
 package object stage {
+
+  // exceptions
+  case class RejectedDepositException(msg: String = "", cause: Throwable = null) extends Exception(msg, cause)
```

Comment:
* I think we should not have a default empty message. You should always report something back to the client.
* Not sure if we would use the `Throwable` at all. From today's discussion I understood the idea was to log everything right way and not propagate all the details upstairs.